src/Symfony/Component/Mime/MessageConverter.php

```diff
@@ -30,6 +30,10 @@ final class MessageConverter
      */
     public static function toEmail(RawMessage $message): Email
     {
+        if ($message instanceof SMimeMessage) {
```

Comment:
Here, we remove the signing/encryption by doing that, which is probably not what I would expect. So, instead, I would throw an exception. I think it makes sense as this class is used to send a message via a provider API, which cannot send a signed/encrypted message.